android/expoview/src/main/java/host/exp/exponent/kernel/Kernel.java

```diff
@@ -741,6 +741,26 @@ public void onEventFailure(String errorMessage) {
     killOrphanedLauncherActivities();
   }
 
+  public static void reloadVisibleExperience(final int activityId) {
```

Comment:
it looks like this needs a @DoNotStrip, and maybe the method below does not (since it's called from this one)